package.json

```diff
@@ -149,7 +149,7 @@
     "cozy-bar": "5.0.7",
     "cozy-client": "1.0.0-beta.14",
     "cozy-client-js": "0.10.0",
-    "cozy-ui": "10.1.0",
+    "cozy-ui": "^10.1.2",
```

Comment:
Out of curiosity, what command do you use to upgrade `cozy-ui` ? It keeps adding the carret which we don't want (renovate will remove it, but still).

Reply:
Next time, can you try to specify the version instead of `latest` please ? I think that will do the trick.

Reply:
Yeah sorry :)